### PR TITLE
docs: Update CONTRIBUTING file

### PR DESCRIPTION
**Pull request by ericfennis**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,14 +21,14 @@ Guidelines for pull requests:
...
-If the icon has not already been requested, [create an issue](https://github.com/colebemis/feather/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
+If the icon has not already been requested, [create an issue](https://github.com/featherity/featherity/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
 
 ## Bug Reports
 
 Before reporting an issue, please search to see if someone has filed a similar issue before. If there is already an open issue, please add a :+1: and/or leave a comment with additional information.
 
 When creating a new issue make sure to include the following:
-- Version of `Feather` in use. Are you running from source/master? Are you using a released build? Which release?
+- Version of `Featherity` in use. Are you running from source/master? Are you using a released build? Which release?
```
Comment (locness3) [inline CONTRIBUTING.md:31]: This might be confusing to people downloading the icons from the website or using the Figma library.

**Comment by jletey** [comment]
Yeah I agree with @locness3, I think we should completely overhaul the contributing file rn, and only focus on designing the icons and opening PRs.

Later when we publish all the proper libraries, then we can add this rhetoric back in.

**Comment by ericfennis** [comment]
Yeah good one, I will remove those lines for now.

**Comment by ericfennis** [comment]
I removed the whole bug reporting section. I think the file is fine for now, and in the future we can write a better one fitting our community.

**Pull request merged**